rskj-core/src/main/java/org/ethereum/config/net/RegTestConfig.java

```diff
@@ -36,10 +39,19 @@ public static RegTestConfig getDefaultRegTestConfig() {
         RegTestConfig config = new RegTestConfig();
 
         config.add(0, new RegTestOrchidConfig());
```

Comment:
I'm not sure, but I think you should follow the same pattern you used for mainnet here:
```suggestion
         BlockchainConfig defaultConfig = new RegTestOrchidConfig();
         defaultConfig.genesisFederation = config.getCommonConstants().getBridgeConstants().getGenesisFederation();
         config.add(0, defaultConfig);
```
Again, not entirely sure about it.